backend/controllers/AuthController.ts

```diff
@@ -13,29 +14,76 @@ export class AuthController extends BaseController {
 
   protected initializeRoutes(): void {
     this.router.get('/siwf', this.getSiwf.bind(this));
-    this.router.get('/account', validateAuthToken, this.getAccount.bind(this));
+    this.router.get('/account', this.getAccount.bind(this));
     this.router.post('/login', this.postLogin.bind(this));
     this.router.post('/logout', validateAuthToken, this.postLogout.bind(this));
   }
 
-  public getSiwf(_req: Request, res: Response) {
-    const payload = AuthHandler.getSiwfRequestConfig();
-    res.send(payload).status(200).end();
+  /**
+   * Retrieves the SIWF configuration and sends the response to the client.
+   * @param req - The request object, not used in this method.
+   * @param res - The response object.
+   */
+  public async getSiwf(_req: Request, res: Response) {
+    const payload = await AccountService.getInstance().then((service) => service.getSWIFConfig());
+    if (!payload) {
+      res.status(HttpStatusCode.InternalServerError).send('Failed to get siwf config').end();
+      return;
+    }
+    res
+      .status(HttpStatusCode.Ok)
+      .send({
+        siwfUrl: payload.siwfUrl,
+        nodeUrl: payload.frequencyRpcUrl,
+        ipfsGateway: Config.instance().ipfsGatewayUrl,
+        providerId: payload.providerId,
+        schemas: [1, 2, 3, 4, 5, 6, 8],
+        network: Config.instance().chainType,
```

Comment:
At some point I'd like to review what exactly is really needed in this response. Okay for now.

Reply:
#43